src/ast/passes/semantic_analyser.h

```diff
@@ -14,6 +14,8 @@
 namespace bpftrace {
 namespace ast {
 
+#define MAX_VARGS 128
```

Comment:
Quoting the Cpp Core Guidelines: "Don’t use macros for constants"
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es31-dont-use-macros-for-constants-or-functions

But also, is there any reason we need to define a maximum number of var args? Could `check_varargs` just default to having its third parameter be `std::numeric_limits<size_t>::max()`?

Reply:
Sure, I can do that. I believe I just copied the 128 from the check_varargs(...) call for "printf", "system", "cat", etc. since I wasn't sure whether or not we had a reason to limit the number of arguments.